inference-engine/samples/common/utils/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+set(TARGET_NAME "samples_utils")
+
+file(GLOB_RECURSE SOURCES "*.cpp" "*.hpp")
+source_group(TREE ${CMAKE_CURRENT_SOURCE_DIR} FILES ${SOURCES})
+
+add_library(${TARGET_NAME} STATIC ${SOURCES})
+set_target_properties(${TARGET_NAME} PROPERTIES FOLDER "src")
+
+# FIXME: need to fix warnings on android
+#include(compile_options)
+#enable_warnings_as_errors(${TARGET_NAME} WIN_STRICT)
+
+ie_developer_export_targets(${TARGET_NAME})
```

Comment:
not available in OpenVINO release package.

Reply:
move to `openvino_developer_export_targets(COMPONENT openvino_common TARGETS format_reader ..)`

Reply:
Added https://github.com/openvinotoolkit/openvino/pull/4167/files#diff-72589b94b3e5aeacbc26e04d16dfd6692aa622cd77b8f09e146b85bb1369775aR183